pkg/configuration/configuration_test.go

```diff
@@ -294,9 +294,10 @@ func TestForbiddenUsernamePrefixesHaveCorrectDefaults(t *testing.T) {
 	defer restore()
 
 	config := getDefaultConfiguration(t)
-	require.Len(t, config.GetForbiddenUsernamePrefixes(), 4)
+	require.Len(t, config.GetForbiddenUsernamePrefixes(), 5)
```

Comment:
we could use the `ElementsMatch` func instead:
```
require.ElementsMatch(t, config.GetForbiddenUsernamePrefixes(), []string{"openshift", "kube", "default", "redhat", "sandbox"})
```

Reply:
Thanks for the recommendation - will change it next time :-)